### PR TITLE
DUM-710 update docs to contain resources info

### PR DESCRIPTION
**Pull request by aandrzejewski**

### CODE REVIEW

File: docs/dabp/guides/functional/manage-resource-via-ui.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Create resources
+*For technical documentation, see [How to create a new resource via the API](../technical/create-resource-via-api.md).*
+
+You can create a new resource using the "New Resource" button on the page. You have to select resource type and give the resource a unique name.
+A new resource is automatically added to the root group. 
+Thus a superuser with the permission `Assign resources to group members` on the root level can assign it to other groups.
+
+## Edit resources
+You can edit resources by clicking on resource on the resource overview. 
+You can then click on the vertical ellipsis button and select "Edit resource".
```
Comment (rvroon) [inline docs/dabp/guides/functional/manage-resource-via-ui.md:22]: Mention that only the name can be changed...

File: docs/dabp/guides/technical/create-resource-type-via-api.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+# Creating resource type via API
+Resource types can be created only via API as they should not change after initial setup of the platform.
```
Comment (rvroon) [inline docs/dabp/guides/technical/create-resource-type-via-api.md:2]: "can only be created"

File: docs/dabp/guides/technical/create-resource-type-via-api.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+Resource type can be linked to a policy. If a resource type is linked to policy, then all resources of that type can be assigned only to groups/members if they also have this policy assigned.
+You can think that a policy on a resource type acts as a filter to whom a resources of this type will be available.
+If resource type is created without a policy id, then all resources of this type will be available to all groups and members 
+
+To create a resource type a POST request must be sent to `https://dabp.onegini.com/delegation/api/v2/resource-types` with a proper request body
+```
+{
+    "name": "Life portfolio type",
+    "policyId": <policy id required to access resources of this type [optional]>
```
Comment (rvroon) [inline docs/dabp/guides/technical/create-resource-type-via-api.md:18]: required and optional in the same line looks strange, maybe change to "needed"

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# In-depth view on resources
+A resource represents the fine-grained access or privileges a user has outside Delegated Administration. 
+For example, the resource can represent a portfolio for a specific reseller.
+It can represent a portfolio of, for instance, mortgages for a specific reseller when a resource is linked to a policy for mortgages.
+Each resource has a specific type which describes if resource is linked with a policy, and what kind of permissions user may have on the resource.
+
```
[the remark this answers is not in the document]
Comment (rvroon) [reply docs/dabp/guides/concepts/resources-in-depth.md:6]: Privileges might need their own subsection

File: docs/dabp/guides/technical/create-resource-via-api.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
+# Creating resource via API
+Resources can get created either [through the UI](../functional/manage-resource-via-ui.md) or through the API.
+To create a resource you first need to [create a resource type via api](./create-resource-type-via-api.md).
+This page is describing how the resource API is used.
+
+## How to create a resource
+In this guide, we will assume Onegini Identity Cloud is available under the URL: `https://dabp.onegini.com`.
+The API is protected with OIDC, so you need to provide an access token to make a successful request.
+Please note that the access token must be linked to a person who has the `Add/edit/delete resource and resource types` permission
```
Comment (rvroon) [inline docs/dabp/guides/technical/create-resource-via-api.md:9]: How do we do this for `non-personal requests`?
Which we now also support I guess...

File: docs/dabp/guides/technical/create-resource-via-api.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+To create a resource a POST request must be sent to `https://dabp.onegini.com/delegation/api/v2/resources` with a proper request body
+```
+{
+    "name":"SELL_LIFE_INSURANCE"
+    "resourceTypeId": <id of existing resource type>
+}
+```
+
+All new resources will automatically get added to the root group.
```
Comment (rvroon) [inline docs/dabp/guides/technical/create-resource-via-api.md:20]: explain why?

File: docs/dabp/guides/technical/person-report.md [hunk 7]
```diff
@@ -1,40 +1,42 @@
 # Person Report
-The Person Report is a record describing all groups a person is a member of.
-This includes all policies he has been assigned to in those groups.
 
-The report can be fetched via an API call.
-The details can also be returned as JWT claim (OIDC) and as SAML attribute.
+The Person Report is a record describing all groups a person is a member of. This includes all policies, resources and resource privileges he has been assigned
+to in those groups.
```
Comment (rvroon) [inline docs/dabp/guides/technical/person-report.md:4]: is a return required?

**Review by rvroon — CHANGES_REQUESTED** [review]

File: docs/dabp/guides/concepts/resources-in-depth.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:6]: fixed

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+## Relation between groups, persons, policies and resources
+Both groups and persons can have resources assigned to it. The difference is that a person can only have resources assigned that are also assigned to a group.
```
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:8]: fixed

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+This means that assigning resources to a person is a contextual operation.
+Let's take this scenario:
+
+- There are 2 resources `Life insurance portfolio` and `Car insurance portfolio`. 
+- There are  2 groups `Life` and `All products`.
+- There is a policy `Sell life insurance` linked to the `Life` and `All products` groups. 
+- Resource `Life insurance portfolio` is linked to group `Life` and `All products`.
+- Resource `Life insurance portfolio` is linked to policy `Sell life insurance`.
```
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:16]: fixed

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+- Resource `Car insurance portfolio` is linked only to group `All products`
```
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:17]: Fixed

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+- Resource `Car insurance portfolio` is not linked to any policy.
```
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:18]: fixed

File: docs/dabp/guides/technical/create-resource-type-via-api.md (hunk 2 above)
Comment (aandrzejewski) [reply docs/dabp/guides/technical/create-resource-type-via-api.md:2]: fixed

File: docs/dabp/guides/technical/create-resource-type-via-api.md (hunk 3 above)
Comment (aandrzejewski) [reply docs/dabp/guides/technical/create-resource-type-via-api.md:18]: fixed

File: docs/dabp/guides/technical/create-resource-via-api.md (hunk 5 above)
Comment (aandrzejewski) [reply docs/dabp/guides/technical/create-resource-via-api.md:9]: fixed - added link to page where we describe m2m calls

File: docs/dabp/guides/technical/create-resource-via-api.md (hunk 6 above)
Comment (aandrzejewski) [reply docs/dabp/guides/technical/create-resource-via-api.md:20]: fixed

File: docs/dabp/guides/technical/person-report.md (hunk 7 above)
Comment (aandrzejewski) [reply docs/dabp/guides/technical/person-report.md:4]: fixed

File: docs/dabp/guides/concepts/resources-in-depth.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+## The hierarchal relationship between groups, subgroups, and resources
+It is not possible to assign more resources to a subgroup than its parent. This means that the DABP tool enforces that a superuser can only assign resources to groups if the parent also has that resource. 
+
+### Adding resource to a group
+Adding a resource to a group does **not** assign this resource to all subgroups, it only makes it possible to assign this resource also to the subgroups. 
+Also, Adding a resource to a group does **not** assign it to all the group members, it only makes it possible to assign the resource to the group members.
+
+### Removing 
+Removing a resource from a parent will result in the removal of the resource for all subgroups and members because the allowed resources are restricted by the resources assigned to the parent group.
```
[the remark this answers is not in the document]
Comment (aandrzejewski) [reply docs/dabp/guides/concepts/resources-in-depth.md:50]: Fixed

File: docs/dabp/guides/functional/manage-resource-via-ui.md (hunk 1 above)
Comment (aandrzejewski) [reply docs/dabp/guides/functional/manage-resource-via-ui.md:22]: Fixed

**Review by rvroon — APPROVED** [review]

**Pull request merged**